src/foobar/api.py

```diff
@@ -23,12 +23,23 @@ def get_card(card_id):
         return None
 
 
-def get_account(card_id):
+def get_account(account_id):
+    try:
+        return Account.objects.get(id=account_id)
+    except Account.DoesNotExist:
+        return None
+
+
+def get_account_card(card_id):
```

Comment:
I would argue that the name is a bit inaccurate. It sounds like the function is supposed to return the card that is associated with given account, while it is doing the opposite. Maybe `get_account_for_card` or `get_account_by_card` instead?